File: Tests/Specs/Helpers/Actors.cs
using Figaro;

namespace Specs.Helpers {

    public class TestStatic {
        public static object TestMethod { get { return null; } }
    }

    public class Actors {

        public static HttpFixture HttpFixture { get { return new HttpFixture {
            Method = "GET",
            Host = "host",
            Uri = "uri",
        };}}
    }
}

File: Tests/RSpecs/specs/helpers/actors.rb
require File.join(File.dirname(__FILE__), "../../../Specs/bin/debug","Figaro.dll")

class Actors

  def self.http_fixture
    http_fixture = Figaro::HttpFixture.new
    http_fixture.Method = "GET"
    http_fixture.Host = "host"
    http_fixture.Uri = "uri"
    http_fixture.Authorization = "BASIC"
    http_fixture.UserName = "lola"
    http_fixture.Password = "run"
    return http_fixture
  end
end